readme.md

```diff
@@ -44,7 +44,7 @@ Yields:
 
 ## API
 
-### visit([node](https://github.com/wooorm/unist#unist-nodes)\[, type\], callback\[, reverse\])
+### visit([node](https://github.com/wooorm/unist#unist-nodes)\[, type], callback\[, reverse])
```

Comment:
Really?  There are four cases, I think (I might be missing some though):

* A reference node without definition node, no need to escape;
* A reference node and a definition node (meant to be linked), no need to escape;
* A reference in `text`, without definition node, no need to escape (currently is);
* A reference in `text`, a definition node, needs escaping (maybe only the definition node);

This does keep all definitions in text escaped though.
Sorry, haven’t put much thought into this, so agina, I might be missing something!